cmd/bm-server/processor/retry_queue.go

```diff
@@ -42,10 +48,46 @@ func ProcessRetryQueue(forceRetry bool) {
 	for _, info := range retryQueue {
 		if info.Retries > MaxRetries {
 			// @TODO: We should send a message back to the user?
+			logrus.Trace("Send a postmaster message")
```

Comment:
could we move the "send message" functionality to a separate function? This keeps the complexity lower and we might be able to use it for other purposes later too

Reply:
absolutely